src/pages/users/UsersPage.tsx

```diff
@@ -302,6 +347,23 @@ const UsersPage: FunctionComponent = () => {
                         </Button>
                     </DialogActions>
                 </Dialog>
+
+                <Dialog open={show} onClose={() => handleDeletion(false)}>
+                    <DialogTitle>{buildTitle(rowsSelection)}</DialogTitle>
+                    <DialogContent>
+                        <DialogContentText>
+                            {buildContent(rowsSelection)}
+                        </DialogContentText>
+                    </DialogContent>
+                    <DialogActions>
+                        <Button onClick={() => handleDeletion(false)}>
+                            <FormattedMessage id="cancel" />
+                        </Button>
+                        <Button type="submit" onClick={deleteUsers}>
+                            <FormattedMessage id="ok" />
+                        </Button>
+                    </DialogActions>
+                </Dialog>
```

Comment:
UserPage was a big componant ... and is now more bigger.
We should split it in several part, like ProfilePages. Maybe not in this PR.

Reply:
the global naming in GridSuite is more my-component.tsx rather than MyComponent.tsx

Reply:
ok, thanks for DeleteUserDialog